.env.test

```diff
@@ -7,4 +7,7 @@ FLASK_SKIP_DOTENV=True
 SECRET_KEY=secretkey
 API_KEY=12345abcde
 API_TOKEN=1234567890
-BOARD_ID=61beadyeyes40
\ No newline at end of file
+BOARD_ID=61beadyeyes40
```

Comment:
You can delete the trello stuff (leave SECRET_KEY, you'll need it in the next module)

Can you update .env.template too?